templates/zerver/api/changelog.md

```diff
@@ -0,0 +1,78 @@
+# Changelog
+
+This page documents changes to the Zulip Server API over time.
+
+The recommended way for a client like the Zulip mobile or desktop apps
+that needs to support interaction with a wide range of different Zulip
+server versions is to check the `zulip_feature_level` parameter in the
+`/register` and `/server_settings` responses to determine which of the
+below features are supported.
+
+## Changes in Zulip 2.2
+
+**Not yet stabilized**:
+
+* [`POST /register`](/api/register-queue): Added `slim_presence`
+  parameter.  Still under development.
+
+**Feature level 2**:
+
+* [`POST /messages/{message_id}/reactions`](/api/add-emoji-reaction):
+  The `emoji_type` parameter is optional; the server will guess the
+  emoji_type if it is not specified (checking custom emoji, then
+  unicode emoji for any with the provided name).
+* `reactions` objects returned by the API (both in `GET /messages` and
+  in `GET /events`) now include the user who reacted in a top-level
+  `user_id` field.  The legacy `user` dictionary (which had
+  inconsistent format between those two endpoints) is deprecated.
+
+**Feature level 1**:
+
+* [`GET /server_settings`](/api/server-settings): Added
+  `zulip_feature_level`, which can be used by clients to detect which
+  of the features described in this changelog are supported.
+* [`POST /register`](/api/register-queue): Added `zulip_feature_level`
+  to the response if `zulip_version` is among the requested
+  `event_types`.
+* [`GET /users`](/api/get-all-users): User objects for bots now
+  contain a `bot_owner_id`, replacing the previous `bot_owner` field
+  (which had the email address of the bot owner).
+* [`GET /users/{user_id}`](/api/get-user): Endpoint added.
+* [`GET /messages`](/api/get-messages): Add support for string-format
+  values for the `anchor` parameter, deprecating and replacing the
+  `use_first_unread_anchor` parameter.
+* [`POST /users/me/subscriptions`](/api/add-subscriptions): Replaced
+  `is_announcement_only` boolean with `stream_post_policy` enum for
+  specifying who can post to a stream.
+* [`PATCH /streams/{stream_id}`](/api/update-stream): Replaced
+  `is_announcement_only` boolean with `stream_post_policy` enum for
+  specifying who can post to a stream.
+* [`GET /streams/{stream_id}`](/api/update-stream): Replaced
+  `is_announcement_only` boolean with `stream_post_policy` enum for
+  specifying who can post to a stream.
```

Comment:
These two link to the same place, which says `PATCH`. Is the `GET` something else?

Reply:
Good catch.